### PR TITLE
New locale selector design

### PR DESCRIPTION
**Pull request by soupette**
My PR is a:-
💅 Enhancement
- New locale selector design

Main update on the:
Admin

New locale selector design

### CODE REVIEW

**Comment by lauriejim** [comment]
@soupette Can you link issues related to this PR please.

File: packages/strapi-admin/admin/src/components/CtaWrapper/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/**
+ * CtaWrapper
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function CtaWrapper({ children }) {
```
Comment (Aurelsicoko) [inline packages/strapi-admin/admin/src/components/CtaWrapper/index.js]: CTAWrapper?

**Comment by soupette** [comment]
@lauriejim done

**Comment by lauriejim** [comment]
@soupette Test fail, can you please take a look.

**Pull request merged**